cmd/xl-storage-disk-id-check.go

```diff
@@ -127,8 +127,11 @@ func (p *xlStorageDiskIDCheck) getMetrics() DiskMetrics {
 	}
 
 	// Do not need this value to be cached.
+	diskMetric.TotalTokens = uint32(p.driveMaxConcurrent)
```

Comment:
Let me change it.

Reply:
No, I am interested in knowing what the configured number of tokens is, I don't really care how many tokens are currently available since that's a useless value.

Not having enough tokens is captured via the number of waiting calls for tokens.

Reply:
This would tell us if the disk has "enough tokens" that we expect is it being blocked prematurely. etc. comes to mind.